reactive/containerd.py

```diff
@@ -486,7 +521,7 @@ def publish_version_to_juju():
         return
 
     output = output.decode()
-    ver_re = re.compile(r"\s*Version:\s+([\d\.]+)")
+    ver_re = re.compile(r"\s*Version:\s+v{0,1}([\d\.]+)")
```

Comment:
well, so the versions built from upstream contain a `v`.  And the deb sources don't?  whatever -- handle that